libcontainer/cgroups/fs/apply_raw.go

```diff
@@ -148,6 +145,11 @@ func (m *Manager) Apply(pid int) (err error) {
 			return err
 		}
 		paths[sys.Name()] = p
+
+		if err := sys.Apply(d); err != nil {
+			m.Paths = paths
```

Comment:
Because `m.Paths` was initialized as `nil` when we create the container, but yes we can initialize `m.Paths` and change it directly, I've mended the commit.